src/components/VueFakeInput.vue

```diff
@@ -96,14 +121,23 @@ export default {
       if (this.allowPaste) {
         const pastedValue = event.clipboardData.getData('text/plain');
         const splitValues = pastedValue.split('');
-        const _this = this;
+        let canPaste = true;
 
-        for (let i = 0; i < this.length; i++) {
-          _this.updateInputValue(i, splitValues[i]);
+        if (this.onlyNumber) {
+          const regx = new RegExp(`^\\d{${this.length}}$`);
+          canPaste = regx.test(pastedValue);
         }
 
-        const [lastInput] = this.$refs[`fk_${this.length}`];
-        lastInput.focus();
+        if (canPaste) {
```

Comment:
Hello, sorry for the delay to review your PR. @leolin1229 why check this `canPaste` value if we are already checking `this.allowPaste` ?